### PR TITLE
fix word wrap

### PR DESCRIPTION
**Pull request by sjpaige**
[add right edge](https://airtable.com/appfJZShN8K4tcWHU/tblvBIYoi6TLmdRAv/viwsO84y649cTJfmn/rec8sdCAuaukALMFk?blocks=hide)

### CODE REVIEW

File: src/Components/HousingFinderList.jsx [hunk 1]
```diff
@@ -13,16 +13,16 @@ import { ListItemLink } from "../ui-kit/ListItemLink";
 const HousingFinderList = ({ resources }) => {
   const classes = useResourcesStyles();
   return (
-    <List dense>
+    <List dense style={{ borderStyle: "solid", width: "100%" }}>
```
Comment (stuttskl) [inline src/Components/HousingFinderList.jsx:16]: Is there a reason why the `Organizations for help finding housing` content has a black border and the others don't?

File: src/Pages/DonatePage.jsx [hunk 2]
```diff
@@ -300,8 +308,8 @@ const DonatePage = () => {
               </Typography>
             </AccordionSummary>
             <AccordionDetails>
-              <Typography>
-                You can send us an email at convictionvacation@gmail.com. We will get back to
+              <Typography className={classes.faqDetailsText}>
+                You can send us an email at contact@clearviction.org We will get back to
```
Comment (stuttskl) [inline src/Pages/DonatePage.jsx:312]: add - "...**and** we will get back to you..."

File: src/Components/HousingFinderList.jsx (hunk 1 above)
Comment (sjpaige) [reply src/Components/HousingFinderList.jsx:16]: It was there to check the boundaries and should be removed.

**Comment by sjpaige** [comment]
@stuttskl I addressed those two issues.

File: src/Components/HousingFinderList.jsx [hunk 3]
```diff
@@ -13,16 +13,16 @@ import { ListItemLink } from "../ui-kit/ListItemLink";
...
-        <ListItem key={idx}  divider>
+        <ListItem key={idx} divider>
           <Grid container>
             <Grid item xs={12} sm={6}>
               <Typography>
                 <ExternalLink href={contact.url}>{contact.name}</ExternalLink>
               </Typography>
               <br />
-              <Typography variant="body2">{contact.desc}</Typography>
+              {/* <Typography variant="body2">{contact.desc}</Typography> */}
```
Comment (dylankershaw) [inline src/Components/HousingFinderList.jsx:25]: can this comment be removed?

**Review by dylankershaw — APPROVED** [review]

**Pull request merged**